src/NUnitFramework/TestBuilder.cs

```diff
@@ -179,7 +179,7 @@ public static ITestResult ExecuteWorkItem(WorkItem work)
             // TODO: Replace with an event - but not while method is static
             while (work.State != WorkItemState.Complete)
             {
-#if PORTABLE || NETSTANDARD1_6
+#if PORTABLE
                 System.Threading.Tasks.Task.Delay(1);
 #else
                 Thread.Sleep(1);
```

Comment:
I think it might be worth making a decision on the future of PCL, before we decide that. We previously talked about it targeting netstandard 1.0, but I'm not sure if we have much need to target that over netstandard 1.3 - which would allow us to use `System.Threading` - and then `Thread.Sleep()` across all platforms. 

If we did that - I'd vote for reducing differences between platforms over any minor performance benefit `Task.Delay()` may give us. 😄